keyboards/bandominedoni/bandominedoni/bandominedoni_via.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
via JSON's don't belong in the QMK repo, and need to be PR'ed to `the-via/keyboards` repo.

Reply:
Actually, this JSON file is for [REMAP](https://remap-keys.app/).
Deleted.